pkg/kubelet/server/stats/types.go

```diff
@@ -115,3 +123,23 @@ type MemoryStats struct {
 	// Cumulative number of major page faults.
 	MajorPageFaults *int64 `json:"majorPageFaults,omitempty"`
 }
+
+// VolumeStats contains data about Volume filesystem usage.
+type VolumeStats struct {
+	// Embedded FsStats
+	FsStats
+	// Path is the directory path the Volume is mounted to
+	Path string `json:"path,omitempty"`
+}
+
+// FsStats contains data about filesystem usage.
+type FsStats struct {
+	// Unique identifier for the filesystem device so that FsStats can be aggregated per-device
+	DeviceId string `json:"deviceId"`
```

Comment:
SGTM.  There will not be a sensible way of aggregating the total fs usage for a pod across logs / rootfs / volumes as we will not know if the device is shared.